_wiki/sfmb_betatest_faq.md

```diff
@@ -56,6 +56,22 @@ latex   : false
 - Because game uses Discord Rich Presence to check your Discord ID.
 - It's best way to check user is REAL beta tester in server or not.
 
+### Q. Some game sprites are bugged, what should i do?
+- Game sprites got corrupted by some reason, mostly download issue. Try [This](sfmb://forceupdate)
+- Or put `sfmb://forceupdate` address on your browser. It should fix corrupted files.
+
+### Q. A friend wants to join the server. How can i invite them?
+- You can talk with an Inviter, but is not guaranteed that your friend will be accepted.

```

Comment:
Wrong answer. 
- It's rule violation for Inviter.
- No one can ask invite friend to Inviter.